drivers/can/can.c

```diff
@@ -1235,9 +1235,9 @@ int can_receive(FAR struct can_dev_s *dev, FAR struct can_hdr_s *hdr,
               memcpy(&waitmsg->cm_hdr, hdr, sizeof(struct can_hdr_s));
 
               nbytes = can_dlc2bytes(hdr->ch_dlc);
-              for (j = 0, dest = waitmsg->cm_data; j < nbytes; j++)
+              if (nbytes)
```

Comment:
why check nbytes ? memcpy can handle size == 0

Reply:
Yes, I will delete this condition